crates/noseyparker/data/default/builtin/rules/dropbox.yml

```diff
@@ -0,0 +1,18 @@
+rules:
+
+- name: Dropbox Access Token
+  id: np.dropbox.1
+
+  pattern: |
+    (?x)
+    \b
+    (sl\.[a-zA-Z0-9_-]{140})
```

Comment:
Are these tokens always `sl.` followed by 140 payload characters?

The examples that appear in [this Dropbox documentation page](https://www.dropbox.com/developers/documentation/http/documentation) have shorter lengths sometimes. I'm not sure if those have been truncated for documentaiton purposes, or if the tokens sometimes have different lengths. Additionally, when I search on GitHub, I see some examples where there are up to 151 characters following the `sl.` prefix.

I was using search queries in GitHub similar to these to explore the lengths of things found in the wild:

- Most tokens look to have at least 130 characters: `content:dropbox AND /\b(sl\.[a-zA-Z0-9_-]{130,134})(?:$|[^a-zA-Z0-9_-])/`
- No tokens longer than 151 characters: `content:dropbox AND /\b(sl\.[a-zA-Z0-9_-]{152,160})(?:$|[^a-zA-Z0-9_-])/`

Reply:
```suggestion
    (sl\.[a-zA-Z0-9_-]{130,152})
```